test/verify/check-metrics

```diff
@@ -748,6 +748,13 @@ class TestCurrentMetrics(MachineCase):
         with b.wait_timeout(180):
             b.wait(lambda: float(b.text("#load-avg .pf-l-flex div:first-child").split()[-1].rstrip(',')) < 10)
 
+        # Test link to user services
+        m.execute("su - admin -c 'export XDG_RUNTIME_DIR=/run/user/$(id -u admin); systemd-run --user --collect --slice cockpittest -p CPUQuota=60% --unit cpu-hog dd if=/dev/urandom of=/dev/null'")
```

Comment:
Please make this more robust by naming the unit "cpu-userhog" or similar, to ensure that this won't collide with the system instances from above. (Same for the memory check)